local.go

```diff
@@ -328,24 +334,7 @@ func evaluate(funcs functions, funcInfo *FunctionInfo, evalType EvaluationType,
 }
 
 // update updates the stats given the new evaluation
-func update(location *Location, optLoc *Location, stats *Stats, funcInfo *FunctionInfo, evalType EvaluationType, iterType IterationType, startTime time.Time) {
```

Comment:
Is there a utility in still having update now that it's trivial? Seems like it may go into evaluate.